addons/getLoadoutsFromUnits/functions/fnc_createConfig.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+private _units = [];
+private _breakline = toString [13,10];
+private _structuredText = "class newFaction {";
+private _tab = "    ";
+
+{
+   if (_x isKindOf "CAManBase") then { _units pushBack _x; };
+}forEach get3DENSelected "object";
+
+
+_structuredText = _structuredText + _breakline + _tab + "class Type {";
+
+{
+   private _return = [_x, _breakline, _tab] call FUNC(getLoadoutAndFormat);
+   _structuredText = _structuredText + _breakline + _return;
```

Comment:
could just make an array of all _returns like
`_returns = _units apply {[_x, _breakline, _tab] call FUNC(getLoadoutAndFormat)}`
And then just join with breaklines
`_returns joinString _breakline`